sdk/datafactory/arm-datafactory/tsconfig.json

```diff
@@ -9,11 +9,19 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es6", "dom"],
+    "lib": [
+      "es6",
+      "dom"
+    ],
     "declaration": true,
     "outDir": "./dist-esm",
     "importHelpers": true
   },
-  "include": ["./src/**/*.ts", "./test/**/*.ts"],
-  "exclude": ["node_modules"]
-}
+  "include": [
+    "./src/**/*.ts",
+    "./test/**/*.ts"
```

Comment:
it should have a "./sample-dev/**/*.ts"  if you are using the latest release tool @azure-tools/js-sdk-release-tools

Reply:
It‘s can not generate code if add parameter of "![image](https://user-images.githubusercontent.com/70128817/155046106-8aae3f4a-2fed-4051-aafb-478636402560.png)
"